src/ui-lib/sass/05-components/03-organisms/_list-image-links.scss

```diff
@@ -0,0 +1,49 @@
+.grav-c-list-image-links {
+  $banner-color-dark-grey: #e5e5e5;
+
+  padding: $grav-sp-l $grav-sp-m calc(#{$grav-sp-l} - #{$grav-sp-m});
+  background-color: $banner-color-dark-grey;
+  column-gap: $grav-sp-l;
+  columns: 2;
```

Comment:
Interesting. Using columns on smaller viewports and flex on larger ones. That means the order of the logos changes - using columns they're sorted vertically and with flex horizontally. It's not a problem since the order of the companies is random anyway. I'm therefore happy for this to be left as is. Just wanted to flag it in case it wasn't intentional.

Reply:
Yes, I noticed this myself. I actually did re-ordered the logos taking both flex & column layouts into consideration. If you're happy with it, then so am I.